cartographer_ros/cartographer_ros/bag_reader.cc

```diff
@@ -29,14 +29,12 @@ namespace cartographer_ros {
 
 constexpr char kTfStaticTopic[] = "/tf_static";
 
-std::unique_ptr<tf2_ros::Buffer> ReadTransformsFromBag(
-    const string& bag_filename) {
+void ReadTransformsFromBag(
+    const string& bag_filename, tf2_ros::Buffer* const buffer) {
```

Comment:
Okay, sure. I was mirroring `ReadStaticTransformsFromUrdf`, where this is also called `buffer`.